Userland/Libraries/LibC/scanf.cpp

```diff
@@ -488,6 +488,7 @@ extern "C" int vsscanf(const char* input, const char* format, va_list ap)
                 conversion_specifier = ConversionSpecifier::Unsigned;
                 break;
             case 'x':
+            case 'X':
```

Comment:
You know what they say
> Please add some tests for this.


I was also extremely surprised that `%X` behaves _literally_ the same as `%x` in glibc too, where it doesn't actually care about the casing.

Reply:
(fwiw, this isn't actually in POSIX: https://pubs.opengroup.org/onlinepubs/9699919799/functions/scanf.html)

Reply:
> You know what they say
> > Please add some tests for this.
> 
> 
> I was also extremely surprised that `%X` behaves _literally_ the same as `%x` in glibc too, where it doesn't actually care about the casing.

Same here honestly, I had to triple check to make sure I was getting Punk'd by POSIX